README.md

```diff
@@ -40,6 +40,12 @@ vars:
     microsoft_ads_schema: your_schema_name
     microsoft_ads_database: your_database_name
 ```
+### UTM Auto Tagging Feature
+This package assumes you are manually adding UTM tags to the `final_url` field within the `ad_history` table. If you are leveraging the auto-tag feature within Microsoft Ads then you will want to enable the `microsoft_auto_tagging_enabled` variable to correctly populate the UTM fields within the `stg_microsoft_ads__ad_history` model.
+```yml
+vars:
+    microsoft_auto_tagging_enabled: true # False by default
```

Comment:
should we include package scope in this example config?

Reply:
Definitely. Good call out, I will add that in.